hocr-merge-dc

```diff
@@ -34,7 +34,7 @@ if len(sys.argv)<2:
     sys.exit(1)
 
 dc_doc = etree.fromstring(open(sys.argv[1]).read())
```

Comment:
Yes. And possibly also make sure we [parse as XHTML](https://github.com/tmbdev/hocr-tools/commit/8cf1682e61b034fb8406016a7ac77a07bd4b9a17#diff-305fdec86283ac99b306636017404b40R28).